help_to_heat/frontdoor/schemas.py

```diff
@@ -489,6 +492,7 @@
         loft_insulation_field_no_insulation: _("I have no loft insulation"),
         loft_insulation_field_dont_know: _("I do not know"),
         loft_insulation_field_no_loft: _("No loft"),
+        "I have up to 100mm of loft insulation": _("I have less than or equal to 100mm of loft insulation"),
```

Comment:
I think put the comment on top of here and add a blank line if the formatter allows it - would more clearly delineate between real answers and the ones kept for backwards-compatibility

Reply:
I'd rather keep the main comment on the map as it applies to all pages in general, have added one to this line too. no newline allowed alas